lib/id3c/cli/command/geocode.py

```diff
@@ -159,7 +162,7 @@ def get_geocoded_addresses(*,
     """
     LOG.debug(f"Reading addresses file {filename}")
 
-    address_file_df = load_file_as_dataframe(filename)
+    address_file_df = load_file_as_dataframe(filename, SUPPORTED_EXTENSIONS)
```

Comment:
I'm not sure I agree with adding a dynamic variable for supported extensions here.
I think we should avoid changing functions while moving them.
Also, the extensions argument gives the illusion of being able to handle multiple file extensions, but in reality, we're only handling `tsv`, `csv`, and `xls*`.